src/screens/AudioScreen.tsx

```diff
@@ -215,4 +225,76 @@ const AudioScreen = ({ route, navigation }): JSX.Element => {
   )
 }
 
+const styles = StyleSheet.create({
```

Comment:
I wasn't able to mock the stylesheet so instead of importing I've put it all in the same file for now